packages/react/src/components/header/Header.module.scss

```diff
@@ -28,10 +31,19 @@
   --action-bar-button-size: 64px;
   --action-bar-item-gutter: 12px;
   --action-bar-item-title-font-size: 14px;
-  --action-bar-item-title-font-color: var(--color-black-90);
+  --action-bar-item-title-font-color: var(--header-color);
+  --lang-selector-item-font-color: var(--header-color);
+  --lang-selector-dropdown-background-color: var(--color-white);
+  --nav-link-font-color: var(--header-color);
+  --nav-border-color: var(--color-black-20);
+  --nav-link-hover-color: var(--color-black-90);
+  --nav-link-dropdown-background-color: var(--color-white);
+  --nav-button-background-color: var(--color-black-5);
```

Comment:
Now the dropdown button's icon is dark when it should be light, so I guess make a variable for that too? Or maybe it's reasonable to have it be the same as `--nav-link-font-color` and set it to NavigationLinkDropdown.module.scss class .button? I think it should be very common to have the link and icon colours be the same 🤔